api/v1beta1/machinedeployment_types.go

```diff
@@ -162,6 +162,11 @@ type MachineDeploymentStrategy struct {
 	// MachineDeploymentStrategyType = RollingUpdate.
 	// +optional
 	RollingUpdate *MachineRollingUpdateDeployment `json:"rollingUpdate,omitempty"`
+
+	// Remediation controls how scaling remediations
+	// should occur during the lifecycle of the dependant MachineSets.
+	// +optional
```

Comment:
Is this intentionally only added to the MD (and not also to the MS)?

If yes, then the PR description is outdated
> This PR is a first step to fix that issue by adding a Remediation strategy on both MachineDeployment and MachineSet

Reply:
Yes, this is to make sure consistency across all MachineSet(s) under management and in place propagation doesn't become a problem